zetaclient/chains/solana/rpc/rpc_live_test.go

```diff
@@ -41,6 +45,46 @@ func LiveTest_GetTransactionWithVersion(t *testing.T) {
 	})
 }
 
+func LiveTest_GetTransactionMessage(t *testing.T) {
+	// create a Solana devnet RPC client
+	client := solanarpc.New(solanarpc.DevNet_RPC)
+
+	// program address
+	gateway := solana.MustPublicKeyFromBase58("ZETAjseVjuFsxdRxo6MmTCvqFwb3ZHUx56Co3vCmGis")
+
+	// get all signatures for the address until the first signature
+	sig := solana.MustSignatureFromBase58(
+		"hrjQH7CJgZU675eDbM3JKKf3tAd3AYtKjtpdSN7bHT4FYPDsFKeJq1BMWjjYLsTJVh1xqE4YNBXwAh2sCE4nxUL",
+	)
+
+	txResult, err := client.GetTransaction(context.Background(), sig, &solanarpc.GetTransactionOpts{
+		Commitment:                     solanarpc.CommitmentFinalized,
+		MaxSupportedTransactionVersion: &solanarpc.MaxSupportedTransactionVersion0,
+	})
+	require.NoError(t, err)
+	require.Nil(t, txResult.Meta.Err)
+
+	// parse gateway instruction from tx result
+	inst, err := observer.ParseGatewayInstruction(txResult, gateway, coin.CoinType_Gas)
+	require.NoError(t, err)
+
+	// get the message
+	fmt.Printf("inst: %+v\n", inst)
```

Comment:
Let's use logger not fmt

Reply:
remove this experimental test https://github.com/zeta-chain/node/pull/3231/commits/a43696d72cf8385ecf0b8a708d221d65e3fdbf6f